build/webpack/lib/postcss-plugins.js

```diff
@@ -51,6 +52,11 @@ module.exports.linting = [
 
 
 const standard = [
```

Comment:
Getting a compile loop near the end of the dev starting that loops about 5 times.
https://ibin.co/w800/3n8HgtHpl2OA.jpg

Reply:
fixed by recent commits